rebasehelper/specfile.py

```diff
@@ -331,23 +331,37 @@ def get_not_used_patches(self):
         """
         return self.patches['not_applied']
 
-    def _comment_out_patches(self, patch_num, note=None):
+    def _process_patches(self, comment_out=[], remove_patches=[], disable_inapplicable_patches=None):
         """
-        Comment out patches from SPEC file
+        Comment out and delete patches from SPEC file
 
-        :var patch_num: list with patch numbers to comment out
+        :var comment_out: list with patch numbers to comment out
+        :var remove_patches: list with patch numbers to delete
+        :var disable_inapplicable_patches: boolean value deciding if the inapplicable patches should be commented out
         """
+        note = 'Following patch contains conflicts'
+
         for index, line in enumerate(self.spec_content):
             #  if patch is applied on the line, try to check if it should be commented out
             if line.startswith('%patch'):
                 #  check patch numbers
-                for num in patch_num:
+                for num in comment_out:
                     #  if the line should be commented out
                     if line.startswith('%patch{0}'.format(num)):
                         comment = '# {}\n'.format(note) if note is not None else ''
```

Comment:
Since `note` is now constant, it would be nicer to use the text directly in `comment` variable.